src/nodejs/unit-http/http.js

```diff
@@ -14,7 +14,7 @@ const {
 
 function createServer (options, requestHandler) {
     if (typeof options === 'function') {
-        requestListener = options;
+        requestHandler = options;
```

Comment:
This should be squashed into the first commit. No point having commits that fix your own commits... ;)

Reply:
That does make review harder. I have to track changes across multiple commits that shouldn't actually exist.

I'm reviewing these things as if that's what will actually be merged. So once I'm finished reviewing I know exactly what state it's being merged as.

That's certainly how it's worked in this project and is what I would consider as best practice.

I see three logical changes in the four commits, so I would expect three commits.

Reply:
I think what @ac000 asks is entirely reasonable. I'll fixup that typo fix commit into the original one, but I do tend to work with atomic commits where related things stay together.

Reply:
Typos squashed into the relevant commits